internal/app/machined/pkg/system/services/machined.go

```diff
@@ -75,6 +75,12 @@ var rules = map[string]role.Set{
 	"/resource.ResourceService/Get":   role.MakeSet(role.Admin, role.Reader),
 	"/resource.ResourceService/List":  role.MakeSet(role.Admin, role.Reader),
 	"/resource.ResourceService/Watch": role.MakeSet(role.Admin, role.Reader),
+	"/cosi.resource.State/Create":     role.MakeSet(role.Admin, role.Reader),
```

Comment:
create for `os:reader`?

Reply:
yep, it will be denied in the resource filter; this line basically says "allow with any role", but on other hand probably better to remove reader here

Reply:
yep, fixed them all, thanks